docs/backup/08-01-create-backup.md

```diff
@@ -18,7 +18,7 @@ Follow the steps to back up Kyma.
     Manual backup
     </summary>
 
-To create a manual backup, use the Backup custom resource. Deploy the following CR to the `kyma-system` Namespace to instruct the Velero server to create a backup. Make sure the indentation is correct.
+To create a manual backup, use the [Backup](https://velero.io/docs/v1.1.0/api-types/backup/) custom resource. Deploy the following CR to the `kyma-system` Namespace to instruct the Velero server to create a backup. Make sure the indentation is correct.
```

Comment:
Could you please refer to the docs from `v1.0.0`? We're using that version

Reply:
Done :)